### ISSUE TITLE
hardsigmoid cuda_dispatch_ptr INTERNAL ASSERT FAILED

### ISSUE THREAD

**Issue by dizcza** [pytorch/pytorch#38007]
## 🐛 Bug

`F.hardsigmoid()` does not work with cuda tensors

## To Reproduce

```python
import torch
torch.nn.functional.hardsigmoid(torch.randn(5).cuda())
```
```
RuntimeError                              Traceback (most recent call last)
<ipython-input-1-4519d961c4ef> in <module>()
      1 import torch
----> 2 torch.nn.functional.hardsigmoid(torch.randn(5).cuda())

/usr/local/lib/python3.6/dist-packages/torch/nn/functional.py in hardsigmoid(input, inplace)
   1586     if inplace:
   1587         return torch._C._nn.hardsigmoid_(input)
-> 1588     return torch._C._nn.hardsigmoid(input)
   1589 
   1590 

RuntimeError: cuda_dispatch_ptr INTERNAL ASSERT FAILED at /pytorch/aten/src/ATen/native/DispatchStub.h:78, please report a bug to PyTorch. DispatchStub: missing CUDA kernel
```

## Expected behavior
A cuda tensor with 5 elements, hardsigmoid applied.

## Environment
PyTorch version: 1.5.0+cu101
Is debug build: No
CUDA used to build PyTorch: 10.1

OS: Ubuntu 18.04.3 LTS
GCC version: (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0
CMake version: version 3.12.0

Python version: 3.6
Is CUDA available: Yes
CUDA runtime version: 10.1.243
GPU models and configuration: GPU 0: Tesla K80
Nvidia driver version: 418.67
cuDNN version: /usr/lib/x86_64-linux-gnu/libcudnn.so.7.6.5

Versions of relevant libraries:
[pip3] numpy==1.18.3
[pip3] torch==1.5.0+cu101
[pip3] torchsummary==1.5.1
[pip3] torchtext==0.3.1
[pip3] torchvision==0.6.0+cu101
[conda] Could not collect

## Additional context
Google colab, GPU instance.

cc @ezyang @gchanan @zou3519 @ngimel @albanD @mruberry

**Comment by zou3519** [comment]
Marking high-pri tentatively for internal assert

**Comment by zou3519** [comment]
So here's the breakdown of what is going on here:
- This operation works on CPU, on 1.5.0
- This operation fails on CUDA, on 1.5.0
- This operation works on both CPU and CUDA on master. https://github.com/pytorch/pytorch/pull/36351 is the PR that fixed this.

I'll leave this issue open just in case we want to add it to a potential 1.5.1 milestone. That would involve cherry-picking and testing #36351. If we don't want that, then please feel free to close the issue.

**Comment by gchanan** [comment]
closing this, since the milestone label is there we won't forget it if we do a 1.5.1.

**Comment by gchanan** [comment]
this isn't a regression (the op didn't exist in 1.4.0), so I don't think we need this in 1.5.1.

**Comment by dizcza** [comment]
I still experience the issue in 1.5.1+cu101.

**Comment by zou3519** [comment]
@dizcza We've fixed this on master. The bugfix will be included as a part of our 1.6 release. As a workaround, you can download a nightly binary until the release
